lib/inspec/utils/telemetry/data_series.rb

```diff
@@ -0,0 +1,45 @@
+require 'json'
+
+# A minimal Dataseries Object
+# Stores the name of the data series and an array of data.
+# Stored data should be a object that supports #to_s
+module Inspec::Telemetry
+  class DataSeries
+    def initialize(name)
+      @name = name
+      @enabled = true
+    end
+
+    attr_reader :name
+
+    def data
+      @data ||= []
+    end
+
+    # This needs to also be set by configuration.
+    def enabled?
+      @enabled
+    end
+
+    def disable
+      @enabled = false
+    end
+
+    def <<(appending_data)
+      data << appending_data
+    end
+
+    alias push <<
+
+    def to_h
```

Comment:
We'll need to check with the real-world endpoints to verify this is the structure they want.  { name1 => [data1], name2 => [data2] } might also be plausible.